k8s/postgresql/chart/postgresql/templates/application.yaml

```diff
@@ -45,9 +45,7 @@ spec:
       app.kubernetes.io/name: "{{ .Release.Name }}"
   componentKinds:
   - group: apps/v1beta2
-    kind: Deployment
-  - group: v1
-    kind: PersistentVolumeClaim
```

Comment:
Following https://github.com/GoogleCloudPlatform/click-to-deploy/pull/600#discussion_r298624035. Should we restore PVC?